pkg/kubelet/kubelet_pods.go

```diff
@@ -2071,6 +2071,18 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 		if oldStatus.Resources == nil {
 			oldStatus.Resources = &v1.ResourceRequirements{}
 		}
+
+		convertCustomResources := func(inResources, outResources v1.ResourceList) {
+			for extendedResourceName, extendedResourceQuantity := range inResources {
```

Comment:
resourceName, resourceQuantity sounds more appropriate instead of extendedResourceName, extendedResourceQuantity